Donation-platform/src/main/java/com/google/DbSetUpUtils.java

```diff
@@ -0,0 +1,128 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google;
+
+import java.util.Collection;
+import java.util.Collections;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.lang.Double;
+import java.util.Set;
+import java.util.HashSet;
+import java.util.HashMap;
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Key;
+import com.google.appengine.api.datastore.KeyFactory;
+import com.google.model.Charity;
+import com.google.model.Tag;
+
+public final class DbSetUpUtils {
+
+    private DatastoreService ds;
+    private DbCalls db;
+
+    //constructor for db setup
+    public DbSetUpUtils() {
+        this.ds = DatastoreServiceFactory.getDatastoreService();
+        this.db = new DbCalls(ds);
+    }
+
+    //getter
+    public DbCalls getDbCalls() {
+        return this.db;
+    }
+
+    //add hardcoded charities into the db
+    public void dbSetUp() {
```

Comment:
rename to populateDatabase or similar

Reply:
Call this in the constructor so that the client does not have to remember to call it

Reply:
I don't want to call it in the constructor because populating the database should only happen once, not every time I create an instance of this class.